src/main/java/ch/admin/bar/siardsuite/database/DbmsRegistry.java

```diff
@@ -160,7 +163,7 @@ public class DbmsRegistry {
 
             ServerBasedDbms.builder()
                     .name("Microsoft SQL Server")
-                    .id("mssql")
+                    .id("sqlserver")
```

Comment:
This incorrect string is causing the issue. The remaining changes are made solely to increase test coverage.